server/src/test/java/org/opensearch/index/IndexSettingsTests.java

```diff
@@ -607,8 +607,12 @@ public void testTranslogGenerationSizeThreshold() {
         assertEquals(actual, settings.getGenerationThresholdSize());
     }
 
+    /**
+     * Test private setting validation for private settings defined in isPrivateSetting()
+     * https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/common/settings/IndexScopedSettings.java
+     */
```

Comment:
GitHub links are ephemeral in that if the path ever changes the link becomes dead. I recommend using a javadoc link instead:
```suggestion
    /**
     * Test private setting validation for private settings defined in {@link IndexScopedSettings#isPrivateSetting}
     */
```